plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -1229,6 +1226,68 @@ var _ = Describe("validator", func() {
 						Expect(err).ToNot(HaveOccurred())
 					})
 				})
+				Context("HA shoot scheduling checks on seed", func() {
+					It("should reject scheduling of multi-zonal HA shoot on non multi-zonal seeds", func() {
+						Expect(coreInformerFactory.Core().InternalVersion().Projects().Informer().GetStore().Add(&project)).To(Succeed())
+						Expect(coreInformerFactory.Core().InternalVersion().CloudProfiles().Informer().GetStore().Add(&cloudProfile)).To(Succeed())
+						Expect(coreInformerFactory.Core().InternalVersion().Seeds().Informer().GetStore().Add(&seed)).To(Succeed())
+						Expect(coreInformerFactory.Core().InternalVersion().SecretBindings().Informer().GetStore().Add(&secretBinding)).To(Succeed())
```

Comment:
These four lines could be extracted in a `BeforeEach()` call so that they are not repeated for every test, right?

Reply:
This change was included in #6530 and relevant tests were written there. Changes were merge to master